dodola/tests/test_services.py

```diff
@@ -32,6 +32,18 @@ def _datafactory(x, start_time="1950-01-01"):
     return out
 
 
+@pytest.fixture(scope="function")
```

Comment:
Very minor thing. I think these fixtures are function-scope by default so you only need `@pytest.fixture`, but this is fine. We can roll with this.